tests/loan-reservation-consistency.unit.php

```
@@ -0,0 +1,85 @@
+<?php
+declare(strict_types=1);
+
+$root = dirname(__DIR__);
+
+function readFileOrFail(string $path): string
+{
+    $contents = file_get_contents($path);
+    if ($contents === false) {
+        fwrite(STDERR, "Cannot read {$path}\n");
+        exit(1);
```

PHPMD (2.15.0)

[error] The function readFileOrFail() contains an exit expression. (undefined)

(ExitExpression)